pkg/scan/transformer.go

```diff
@@ -38,64 +38,68 @@ func NewTransformer(clock Clock) Transformer {
 	}
 }
 
-func (t *transformer) Transform(artifact harbor.Artifact, source trivy.ScanReport) (target harbor.ScanReport) {
-	var vulnerabilities []harbor.VulnerabilityItem
+func (t *transformer) Transform(artifact harbor.Artifact, source trivy.ScanReport) (harbor.ScanReport) {
+	vulnerabilities := make([]harbor.VulnerabilityItem, len(source.Vulnerabilities))
```

Comment:
Pre-allocate memory so that the array doesn't get resized.

Reply:
NIT: Remove redundant parentheses for the return type `(harbor.ScanReport)`